<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enfermedades Tropicales</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>¿Cuál de los siguientes síntomas presenta?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>¿Ha estado recientemente en...?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnviarBTN">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enfermades Tropicales</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="RespuestaLBL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Respuesta!</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>¿Recientemente... ?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CausasCMB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>He tenido relaciones sexuales</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>He sido picado por un zancudo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hen recibido una transfusion de sangre</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="PaisCMB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>America Sur</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>America Norte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>America Central</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Africa</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="SintomasLW">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>Fiebre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dolor Muscular</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Perdida de apetito</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vomitos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Conjuntivitis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cansancio</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ictericia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sangrado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arritmia</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
